test/black-box/ListingItemPostCommand.test.ts

```diff
@@ -8,12 +8,8 @@ import { ListingItemTemplateCreateRequest } from '../../src/api/requests/Listing
 
 describe('ListingItemPostCommand', () => {
     const testUtil = new BlackBoxTestUtil();
-    const listingItemTemplateService = null;
-    const messageBroadcastService = null;
-    const itemFactory = null;
-    const marketService = null;
     const listingItemService = null;
-    const method =  new ListingItemPostCommand(listingItemTemplateService, messageBroadcastService, itemFactory, marketService, null, Logger).name;
+    const method =  new ListingItemPostCommand(listingItemService, Logger).name;
     const addMakretMethod =  new MarketCreateCommand(listingItemService, Logger).name;
```

Comment:
method should be a string matching the command.name, 'postitem' in this case.

export const rpc = async (method: string, params: any[] = [] ): any => {
    const body = { method, params, jsonrpc: '2.0' };
    return await api('POST', '/api/rpc', { body });
};